### ISSUE TITLE
X-Frame-Options nginx headers

### ISSUE THREAD

**Issue by Matt-Yorkley** [openfoodfoundation/ofn-install#107]
Can we remove the X-Frame-Options header from the nginx settings? 

Embedded Shopfronts needs to be able to turn this header on and off for embedded content to function. We currently set the header to `DENY` by default from within Rails, and remove the header when serving an embedded page.

If it's set by nginx it can't be changed from within Rails.

**Comment by enricostano** [comment]
LOL, with @danypr92 we were just looking at that duplicated header 😂 

```
< HTTP/1.1 200 OK
< Server: nginx
< Date: Wed, 17 Jan 2018 12:29:13 GMT
< Content-Type: text/html; charset=utf-8
< Transfer-Encoding: chunked
< Connection: keep-alive
< Status: 200 OK
< Strict-Transport-Security: max-age=31536000
< X-Frame-Options: DENY
< Content-Security-Policy: frame-ancestors 'none'
< X-UA-Compatible: IE=Edge,chrome=1
< ETag: "40ea4daeeebfdcba3dfbca1c10c3e065"
< Cache-Control: max-age=0, private, must-revalidate
< Set-Cookie: _session_id=8cd0b5362616cafa2fa5cacc46799559; path=/; secure; HttpOnly
< X-Request-Id: f6db2eea19c140910d037a06bd4a6029
< X-Runtime: 0.431980
< X-Rack-Cache: miss
< X-Content-Type-Options: nosniff
< X-Xss-Protection: 1; mode=block
< X-Frame-Options: DENY
< Strict-Transport-Security: max-age=31536000
```

**Comment by enricostano** [comment]
@Matt-Yorkley I'm closing this since it has been solved by #121 , right?
